### PR TITLE
time/memory benchmarking

### PR DESCRIPTION
**Pull request by AKS1996**
solves https://github.com/AKS1996/DiffOpt.jl/issues/11
in need some guidance. i'm total noob at benchmarking. but i was thinking to improve both sparsity and time in same PR

### CODE REVIEW

**Comment by AKS1996** [comment]
using this script for julia benchmarking (in need of suggestions :Ddiffer

File: src/MOI_wrapper.jl [hunk 1]
```diff
@@ -43,6 +43,8 @@ const SUPPORTED_VECTOR_SETS = Union{
     MOI.PositiveSemidefiniteConeTriangle,
 }
 
+const SUPPORTED_TERMINATION_STATUS = [MOI.LOCALLY_SOLVED, MOI.OPTIMAL, MOI.ALMOST_OPTIMAL]
```
Comment (matbesancon) [inline src/MOI_wrapper.jl:46]: When do we have cases with `MOI.LOCALLY_SOLVED`?

Comment (AKS1996) [reply src/MOI_wrapper.jl:46]: yep. i remember differentiating a convex QP in some solver gave this result. let me fetch the history

**Comment by matbesancon** [comment]
> `but i was thinking to improve both sparsity and time in same PR`

I would suggest sparsity in a different PR not to get things confused

**Comment by AKS1996** [comment]
> I would suggest sparsity in a different PR not to get things confused

noted will create a new PR

**Comment by matbesancon** [comment]
Your Python benchmark seems to profile memory usage more than time?

**Comment by AKS1996** [comment]
as i said, noob at profiling :sweat_smile:
but i ran them separately (one at a time) so there should'nt be any overlap
although i'm not sure these results are from single or multiple iterations

**Comment by matbesancon** [comment]
https://docs.python.org/3/library/timeit.html
this module should do what you want

**Comment by AKS1996** [comment]
PR doesn't contains any code/contribution for the project 
closing it

**Pull request closed without merging**